### DISCUSSION TITLE
How can you call getHTML(); function on custom button click (not every time the editors content is updated)

### DISCUSSION THREAD

**Discussion by izabayo7** [ueberdosis/tiptap#782] [General Questions] [answered]
I need help 
Is there any way we can only call getJSON() or getHTML() function when a user click a save button instead of calling those function each time the editors content is updated

**Comment by hanspagel** [comment, marked as answer]
Sure! Hope that helps:

https://codesandbox.io/s/vue-issue-template-hjm8q?file=/src/App.vue

in the template:

```html
  <button class="menubar__button" @click="getHTML">
    <span>getHTML</span>
  </button>
```

add a method:

```js
  methods: {
    getHTML() {
      alert(this.editor.getHTML());
    }
  },
```

**Reply by izabayo7** [reply]
Thank you so much
